### PR TITLE
added some templates

### PR DESCRIPTION
**Pull request by psq21**

### CODE REVIEW

File: huh/db.py [hunk 1]
```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
         res = cur.execute(f"SELECT rowid AS id, * FROM {cls.table_name}", ())
 
         return (cls(*row) for row in res)
+    
+    @classmethod
```
Comment (SebassNoob) [inline huh/db.py]: can you move these methods to the Announcement class? 
the Entry class is a base class that contains base methods that all child objects/table have (like selecting by id, getting all the entries in the sql db, etc).

File: huh/db.py [hunk 2]
```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
...
+    def all_ann(cls, conn):
```
Comment (SebassNoob) [inline huh/db.py]: this method is not necessary once you move all the methods to the announcement class, use ``Announcement.all()`` instead

File: huh/db.py [hunk 3]
```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
...
+        cur.row_factory = sqlite3.Row
+        res = cur.execute(
+            """
+            SELECT Announcement.rowid,title,name,timestamp,content FROM 
+            Announcement JOIN User ON Announcement.author_id=User.rowid
+            """
+            )
+        return res.fetchall()  
+    
+    @classmethod
```
Comment (SebassNoob) [inline huh/db.py]: use staticmethod decorator -- classmethod is only for static methods that need access to cls (the  properties on the static class), and cls seems to be unused here

**Review by SebassNoob — CHANGES_REQUESTED** [review]
good job tho code is quite clean

File: huh/db.py (hunk 2 above)
Comment (psq21) [reply huh/db.py]: oh I kinda need the name of the user also tho

Comment (SebassNoob) [reply huh/db.py]: you can get the id of the user on the api layer via ``current_user.id``. the ``@login_required`` decorator must be present (i think)

eg.
```python
from flask_login import login_required, current_user
from flask import render_template, Blueprint, request
from huh.db import Comment, connect

bp = Blueprint("comment", __name__, url_prefix="/comment")

@bp.route("/create", methods=["POST"])
@login_required
def create():
    author_id = current_user.id
    # rest of ur code

```

**Pull request merged**